### PR TITLE
Add a chart of iidm versions and powsybl core versions

### PR DESCRIPTION
**Pull request by EtienneLt**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,3 +159,20 @@ You can revert this configuration with the following command:
 
 ### Usage
 Once the configuration is done, you just need to use `./mvnw` instead of `mvn` in your commands.
+
+### IIDM version vs PowSyBl core version
```
Comment (olperr1) [inline README.md:163]: This should be a subsection of `Using Maven Wrapper`:

```suggestion
## IIDM version vs PowSyBl core version
```

Comment (EtienneLt) [reply README.md:163]: ok

Comment (EtienneLt) [reply README.md:163]: I think it is already because ## Using Maven Wrapper

Comment (olperr1) [reply README.md:163]: Sorry, I mean "it should **not** be"

File: README.md
Comment (flo-dup) [inline README.md:1]: I don't know if you should put the chart at the end. But anyway the whole readme is such a mess that it's ok :tired_face:

**Review by flo-dup — COMMENTED** [review]
Couldn't we use this opportunity to start a new README.md from scratch? What you added is the most interesting info of the whole file. There's almost nothing describing what powsybl-core is and contains, whereas the install section is only for very specific users, and the maven wrapper section could be located in a wiki help page.

**Comment by annetill** [comment]
Solved with information in the wiki https://github.com/powsybl/powsybl-core/wiki

**Pull request closed without merging**